apps/frontend/src/middleware.ts

```
@@ -1,58 +1,74 @@
+  const path = request.nextUrl.pathname;
+  const publicPath = ['/']
+
+  const result = await verifyToken(token || "");
+
+  // If !result and trying to access protected routes
+  if(!result && !publicPath.includes(path)){  
+    return NextResponse.redirect(new URL("/", request.url));
+  }
+
+  // If result, valid user  with a verified profile, trying to access a public path
+  if(result && result.isValidUser && result.userVerification && publicPath.includes(path)){
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)